Frameworks/UIKit/UIApplication.mm

```diff
@@ -423,234 +332,131 @@ + (UIApplication*)sharedApplication {
-            break;
-
-        case UIInterfaceOrientationLandscapeLeft:
-            rect.origin.x = appFrame.origin.y;
-            rect.origin.y = appFrame.origin.x;
-            rect.size.width = appFrame.size.height;
-            rect.size.height = appFrame.size.width;
-
-            [statusBarRotationLayer
-                setCenter:CGPointMake(appFrame.origin.x + appFrame.size.width / 2.0f, appFrame.origin.y + appFrame.size.height / 2.0f)];
-            [popupRotationLayer
-                setCenter:CGPointMake(appFrame.origin.x + appFrame.size.width / 2.0f, appFrame.origin.y + appFrame.size.height / 2.0f)];
-            rect.origin.x = 0.0f;
-            rect.origin.y = 0.0f;
-            [statusBarRotationLayer setBounds:rect];
-            [popupRotationLayer setBounds:rect];
-
-            trans = CGAffineTransformMakeRotation(270.0f / 180.0f * kPi);
-            [statusBarRotationLayer setTransform:trans];
-            [popupRotationLayer setTransform:trans];
-
-            _statusBarInsets.left = statusBarHeight;
-            _statusBarInsets.right = 0.0f;
-            _statusBarInsets.top = 0.0f;
-            _statusBarInsets.bottom = 0.0f;
-            changeHostOrientation = 1;
-            break;
-
-        case UIInterfaceOrientationPortraitUpsideDown:
-            rect.origin.x = appFrame.origin.x;
-            rect.origin.y = appFrame.origin.y;
-            rect.size.width = appFrame.size.width;
-            rect.size.height = appFrame.size.height;
-
-            [statusBarRotationLayer
-                setCenter:CGPointMake(appFrame.origin.x + appFrame.size.width / 2.0f, appFrame.origin.y + appFrame.size.height / 2.0f)];
-            [popupRotationLayer
-                setCenter:CGPointMake(appFrame.origin.x + appFrame.size.width / 2.0f, appFrame.origin.y + appFrame.size.height / 2.0f)];
-            rect.origin.x = 0.0f;
-            rect.origin.y = 0.0f;
-            [statusBarRotationLayer setBounds:rect];
-            [popupRotationLayer setBounds:rect];
-
-            trans = CGAffineTransformMakeRotation(kPi);
-            [statusBarRotationLayer setTransform:trans];
-            [popupRotationLayer setTransform:trans];
-
-            _statusBarInsets.left = 0.0f;
-            _statusBarInsets.right = 0.0f;
-            _statusBarInsets.top = 0.0f;
-            _statusBarInsets.bottom = statusBarHeight;
-            changeHostOrientation = 2;
-            break;
-
-        default:
-            TraceVerbose(TAG, L"Unknown orientation %d", _curOrientation);
-            assert(0);
-            break;
-    }
+- (UIInterfaceOrientation)statusBarOrientation {
+    UNIMPLEMENTED_WITH_MSG("UWP status bar can be directly accessed via WinRT APIs, and will eventually be supported via UIViewController APIs.");
+    return StubReturn();
 }
 
-- (void)_setInternalOrientation:(UIInterfaceOrientation)orientation {
-    _internalOrientation = orientation;
+/**
+ @Status Interoperable
+*/
+- (void)setDelegate:(id)delegateAddr {
```

Comment:
err API coverage data...

---
In reply to: [110412323](https://github.com/Microsoft/WinObjC/pull/2445#discussion_r110412323) [](ancestors = 110412323,110294608)